modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -320,6 +334,18 @@ private CompletableFuture processOperation(
             case ClientOp.TUPLE_GET_AND_DELETE:
                 return ClientTupleGetAndDeleteRequest.process(in, out, ignite.tables());
 
+            case ClientOp.SQL_EXEC:
+                return handler.execute(in, out);
```

Comment:
Can we follow the same pattern as above with one class per request, so the code looks uniform? `ClientSqlRequestHandler` simply wraps `QueryEventHandler`, not sure if we need this class at all.

Reply:
Got rid of ClientSqlRequestHandler. Now it looks like the other cases.